pkgs/games/openspades/default.nix

```diff
@@ -58,7 +58,7 @@ stdenv.mkDerivation rec {
   NIX_CFLAGS_LINK = "-lopenal";
 
   meta = with lib; {
-    broken = stdenv.isDarwin;
+    broken = (stdenv.isDarwin && stdenv.isAarch64) || (stdenv.isLinux && stdenv.isAarch64) || (stdenv.isDarwin && stdenv.isx86_64);  # never built on aarch64-linux since first introduction in nixpkgs
```

Comment:
for example here we definitely want to split the comment into an extra line

Reply:
I agree